README.md

```diff
@@ -125,6 +125,19 @@ import { CHECKABLE_LIST_ITEM } from "draft-js-checkable-list-item"
 ]
 ```
 
+### `entityType`
+Dictionary for `IMAGE` and `LINK` entity.
+
+#### Example:
+
+```js
+const entityType = {
+  IMAGE: "IMAGE",
+  LINK: "LINK",
+}
```

Comment:
This example isn't very helpful :wink:

The docs should explain why this option exists and when you might need to use it, and then the example should show a potential actual usecase.

Reply:
I've updated the `README`, I'm not sure how to be clearer though.

As mentioned in #69, plugins such as `draft-js-image-plugin` and `draft-js-export-markdown` expects an image entity to be `IMAGE` instead of `IMG`.

`IMG` is the default. So because `IMG` is the default, `draft-js-markdown-plugin` won't work "out of the box" with `draft-js-image-plugin` or `draft-js-export-markdown`.